app/services/show_attendee_phone_number_for_user.rb

```diff
@@ -0,0 +1,16 @@
+class ShowAttendeePhoneNumberForUser
```

Comment:
Since it's just view-related, idiomatic rails is to encapsulate this as a [Helper](http://mixandgo.com/blog/the-beginner-s-guide-to-rails-helpers).